client/v2/internal/flags/flags.go

```diff
@@ -0,0 +1,19 @@
+package flags
+
+// This defines flags names that can be used in autocli.
+const (
+	// FlagFrom is the flag to set the from address from which to sign the transaction.
+	FlagFrom = "from"
+
+	// FlagOutput is the flag to set the output format.
+	FlagOutput = "output"
+
+	// FlagNoIndent is the flag to not indent the output.
+	FlagNoIndent = "no-indent"
+)
+
+// List of supported output formats
+const (
+	OutputFormatJSON = "json"
+	OutputFormatText = "text"
```

Comment:
Is it it not actually YAML, from what I remember seeing on the output?

Reply:
Correct, but in this SDK this is what `text` refers to (https://github.com/cosmos/cosmos-sdk/blob/1d4c265782574c7fc4d0724454eeded174e76cbf/client/flags/flags.go#L96)